explainer.md

```diff
@@ -187,6 +187,30 @@ There are several open questions to address:
    source.
 1. How to handle messaging.
 
+## Security and privacy considerations
+
+See [the corresponding section](
+https://w3c.github.io/presentation-api/#security-and-privacy-considerations) of
+the Presentation API spec. Below, we list characteristics unique to site
+initiated mirroring.
+
+### Personally identifiable information
+
+By [monitoring the list of available presentation displays](
+https://w3c.github.io/presentation-api/#monitoring-the-list-of-available-presentation-displays),
+a website can determine whether the user has devices compatible with mirroring,
+which is one new bit of information that can be used for fingerprinting. This
+bit however would likely strongly correlate with whether the user has devices
```

Comment:
I found this sentence hard to follow, especially if the reader doesn't know what the "existing modes" are.

Maybe:

"Nearly all presentation displays support mirroring, so this additional information does not add significantly to the fingerprinting surface of the API."